adrs/8618797096/README.md

```diff
@@ -0,0 +1,61 @@
+# Usage of JSX Fragment
+
+* **State:** Draft
+* **Created:** 2020-11-15
+* **Tags:** jsx
+
+## Context
+
+Today we have a way to specify fragments in our React, Preact codebase or
+any other library that adopts such technology.
+
+1. Using `Fragment` from React import:
+
+    ```tsx
+    import { Fragment } from 'react';
+    function MyApp() {
+      return (
+        <Fragment></Fragment>
+      )
+    }
+    ```
+
+2. Or importing everything under a scope:
+
+    ```tsx
+    import * as React from 'react';
+    function MyApp() {
+      return (
+        <React.Fragment></React.Fragment>
```

Comment:
The difference between options 1 & 2 seems out-of-scope of this ADR.   That difference is a question of how to import named imports.

Reply:
I point this out because ADRs are about evaluating options.  Our options aren't 1 vs 2 vs 3, it's more like (1a or 1b) vs 2

Reply:
> The difference between options 1 & 2 seems out-of-scope of this ADR. That difference is a question of how to import named exports.

The intention is to showcase multiple ways to express JSX Fragments today.

Reply:
> I point this out because ADRs are about evaluating options. Our options aren't 1 vs 2 vs 3, it's more like (1a or 1b) vs 2

I think it is a matter of technicality here, I agree with you, I changed it to bullet point since the numbers are meaningless.